src/TipBox.vue

```diff
@@ -80,10 +85,16 @@
         return this.type === 'none'
       },
       isSeamless() {
-        return toBoolean(this.seamless);
+        return !this.light && this.seamless;
```

Comment:
Just wondering if we should fail hard or emit an error message if both `light` and `seamless` is present. It feels like it might make debugging easier for the user when trying to figure out why the output might not be what he is expecting.

Reply:
> Just wondering if we should fail hard or emit an error message if both `light` and `seamless` is present. It feels like it might make debugging easier for the user when trying to figure out why the output might not be what he is expecting.

I think emitting an error message is a good idea, we could that in the new `componentParser` in MarkBind easily

We could do this in a seperate PR with warnings for deprecated attributes in one go as well, something like `"Add warning messasges for inconsistent component attributes"` perhaps

I don't mind updating https://github.com/MarkBind/markbind/pull/1025 for the **light vs seamless** thing alone first though, let me know which would be better 😄

Reply:
I think it would be better that we write a common utility to check for this. Since both #129 and this have the possibility of inconsistent attributes. I can either include it in my PR, although I'm more inclined that it'd be in a seperate PR altogether.

Reply:
Adding a common utility in a common PR after both these PRs are merged and applying it to all the new changes at once sounds good to me. Let's open an issue to track this.

Perhaps we can leave some `// TODO`s to indicate where these checks should happen to make it easier for the person working on this issue.

Reply:
> Perhaps we can leave some `// TODO`s to indicate where these checks should happen to make it easier for the person working on this issue.

Updated https://github.com/MarkBind/markbind/pull/1025 with it